pyprojectsort/main.py

```diff
@@ -27,6 +27,7 @@ def _read_cli(args: list) -> argparse.Namespace:
         version=__version__,
         help="show package version and exit",
     )
+    parser.add_argument("--check", help="show package version and exit", action='store_true', default=False)
```

Comment:
We can just update the `help` text here - I have simply taken the text from the `black` tool in the attached suggestion. Also, when we set 'action' to 'store_true', the default will automatically be set to 'False' if the option is not provided, so we are good to not set that - just to reduce a small bit of code.

```suggestion
    parser.add_argument(
        "--check",
        help=(
            "Don't write the files back, just return the status. Return code 0 means"
            " nothing would change. Return code 1 means the file would be reformatted"
        ),
        action="store_true",
    )
```

Reply:
I copy-pasted a bit and forgot to fix help, my bad:)

Reply:
Final comment: I see "store_true" is surrounded by single quotes (') in what was pushed. Can you ensure that black and ruff have been run against the codebase and then upload that fix?

Reply:
I've run black and ruff. 

ruff also want me to fix the next: 

- tests/unit/test_main.py:3:1: I001 [*] Import block is un-sorted or un-formatted
- tests/unit/test_main.py:88:9: D107 Missing docstring in `__init__`
- tests/unit/test_main.py:88:54: FBT001 Boolean positional arg in function definition

I decided leave it for now. What do you think? Is it necessary to fix?

Reply:
I think would need to fix for now as pipeline on main would fail after merging. Apologies for only catching this now, I need to amend the pipeline to run on pull requests also.

1. If you run `ruff --fix .`, it will automatically fix the first issue.
2. For the second issue, you can add `"""Initialise test data arguments."""` in __init__ of TestArgs - if that is the location that the linting is referring to.
3. For the third one, on the same line as def __init__ for TestArgs, but 2 spaces after it, add a comment to ignore that rule there e.g. `def __init__(self, file: str = "test_data.toml", check: bool = None):  # noqa: FBT001`